pkg/codegen/internal/test/testdata/external-resource-schema/go/example/argFunction.go

```diff
@@ -7,7 +7,7 @@ import (
 	"context"
 	"reflect"
 
-	"github.com/pulumi/pulumi-random/sdk/v2/go/random"
+	"github.com/pulumi/pulumi-random/sdk/v4/go/random"
```

Comment:
There's some nastiness here.. usually when I get these diffs I re-run with PULUMI_IGNORE_AMBIENT_PLUGINS=1 and I don't get those anymore; however your PR seems to have passed the CI checks so maybe these are the right diffs that have to be included.